wdl/transforms/new-base/src/main/scala/wdl/transforms/base/wdlom2wom/graph/CallElementToGraphNode.scala

```diff
@@ -52,6 +52,15 @@ object CallElementToGraphNode {
         case None => s"Cannot resolve a callable with name ${a.node.callableReference}".invalidNel
       }
 
+    def supplyableInput(definition: Callable.InputDefinition): Boolean = {
+        !definition.isInstanceOf[FixedInputDefinitionWithDefault] &&
+        !definition.name.contains(".") // NB: Remove this check when sub-workflows allow pass-through task inputs
```

Comment:
ToL: Does this deserve/have a ticket, or is it fine if it's never fixed/tracked/seen-again?

Reply:
This is more of a "if WDL 2.0 removes this requirement (🤞) then this line goes away"